server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java

```diff
@@ -609,10 +609,9 @@ private ImmutableMap<Path, String> getScalarAnswers(ApplicantQuestion question)
                                       programDefinition.id(),
                                       URLEncoder.encode(fileKey, StandardCharsets.UTF_8))
                                   .url())
-                  .collect(Collectors.joining(", ", "[", "]")));
+                  .collect(Collectors.joining(", ")));
```

Comment:
What's the reason for this change (removing the brackets). I think other lists that we export in CSV use the bracket notation so I thought it'd be good to be consistent.

Reply:
I was chatting with Matthew in Seattle about the format that would be easiest to parse, and removing the brackets lets you do text-to-columns.

The other thing that uses brackets in this method, the checkbox questions, isn't actually exported that way, and is instead exported as a column for each option.

Reply:
Ah, I didn't realize that. The links also end up formatting properly if you do it this way, which is good.